docker-compose.yml

```diff
@@ -3,11 +3,12 @@ services:
   node:
     image: network-of-terms-comunica:latest
     user: node
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
     restart: always
     volumes:
-      - .:/usr/src/app
-      - /usr/src/app/node_modules
```

Comment:
@sdevalk Not sure why you did this.

Reply:
This 'trick' enables fast development: using this, a change on the host (e.g. because I update a source file) is instantly reflected in the container. See [here](https://jdlm.info/articles/2016/03/06/lessons-building-node-app-docker.html#the-node_modules-volume-trick) for example. However, I'm open to improvement!

Reply:
Ah yes, the `node_modules/` volume makes sure that the `node_modules/` directory that is filled during `docker-compose build` is still available after starting the containers with `docker-compose up`. So reverted.